Vanilla-JS-Projects/Basic/Random-Quote-Generator/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Random Quote Generator</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="heading">
+            <h1 class="main_heading">Random Quote Generator</h1>
+        </div>
+        <div class = "quote_gen">
+            <div class = "box">
+                <h3>Click "Generate" Button to generate a random Quote</h3>
+            </div>
+            <div class="btn">
+                <button class = "btn">Generate</button>
+            </div>
+        </div>
+        <div class="footer">

```

Comment:
Remove this footer section from the project. No need of it for the project `WebMasterLog`.